packages/brick-container/src/loadBootstrapData.ts

```
@@ -174,28 +174,16 @@ function mergeRuntimeSettings(
-    ...settings.misc,
-    ...misc,
-  };
+  settings.featureFlags = { ...settings.featureFlags, ...featureFlags };
+  settings.misc = { ...settings.misc, ...misc };
   Object.assign(settings, rest);
 }
 
 const appRuntimeDataMap = new Map<
   string,
-  Promise<RuntimeMicroAppStandaloneData | undefined>
+  Promise<RuntimeApi_RuntimeMicroAppStandaloneResponseBody | void>
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)